railties/test/generators/generated_attribute_test.rb

```diff
@@ -136,6 +136,10 @@ def test_default_value_is_empty_string
     end
   end
 
+  def test_default_value_is_empty_hash
```

Comment:
The test name references an empty hash, as it looks like it used to be.

Seeing the other tests, it seems like the pattern for the name is something like:
```suggestion
  def test_default_value_is_json
```